core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -63,7 +65,7 @@
     private List<String> portBindings = new ArrayList<>();
 
     @NonNull
-    private String dockerImageName = "alpine:3.2";
+    private Future<String> image;
```

Comment:
The previous field was private, so it's safe to delete it. However, getters/setters are there and not deleted (to keep backward compatibility)

`image` name was selected because it might me both image name ( "owner/repo:tag" ), and image id ( "bde98019a9e2" )